### ISSUE TITLE
0.5.1 ARM binaries: Cannot find libLTO.so

### ISSUE THREAD

**Issue by simonbyrne** [JuliaLang/julia#20936]
I just tried out the ARM 0.5.1 build from julialang.org/downloads/ on my Raspberry Pi, and it fails with
```
├ julia
julia: error while loading shared libraries: libLTO.so: cannot open shared object file: No such file or directory.

├ ldd lib/libjulia.so
	linux-vdso.so.1 (0x7eda0000)
	/usr/lib/arm-linux-gnueabihf/libarmmem.so (0x76bc3000)
	libLLVM-3.9.so => /home/pi/julia/lib/julia/libLLVM-3.9.so (0x7573c000)
	libdl.so.2 => /lib/arm-linux-gnueabihf/libdl.so.2 (0x75714000)
	librt.so.1 => /lib/arm-linux-gnueabihf/librt.so.1 (0x756fd000)
	libpthread.so.0 => /lib/arm-linux-gnueabihf/libpthread.so.0 (0x756d5000)
	libstdc++.so.6 => /home/pi/julia/lib/julia/libstdc++.so.6 (0x7558f000)
	libm.so.6 => /lib/arm-linux-gnueabihf/libm.so.6 (0x75514000)
	libgcc_s.so.1 => /home/pi/julia/lib/julia/libgcc_s.so.1 (0x754e8000)
	libc.so.6 => /lib/arm-linux-gnueabihf/libc.so.6 (0x753a7000)
	/lib/ld-linux-armhf.so.3 (0x54b3f000)
	libLTO.so => not found
	libz.so.1 => /lib/arm-linux-gnueabihf/libz.so.1 (0x75380000)
```

**Comment by simonbyrne** [comment]
Do we need #19144?

**Comment by ViralBShah** [comment]
Same problem on power too.

**Comment by tkelman** [comment]
Does this happen on nightlies?
Can also just go back to 3.8.

**Comment by vchuravy** [comment]
It seems we would need to backport #19054 since power and arm default to 3.9

**Comment by simonbyrne** [comment]
> Does this happen on nightlies?

no

**Comment by ViralBShah** [comment]
It's an easy fix, so let's not go back to 3.8. Can we go to 3.9.1?

**Comment by simonbyrne** [comment]
Can we do something about this soon? If not, can we put the 0.5.0 binaries back up since the current ones are useless.

**Comment by tkelman** [comment]
> back up

They didn't go anywhere. We can change the links for arm and power back to 0.5.0 for now, and make an 0.5.2 at some point with the fix backported. Would appreciate if people who care about these platforms would test things while releases are being prepared, I have no access to either of these architectures.

**Comment by ViralBShah** [comment]
I will arrange to get you access to these architectures. Can we put out a quick 0.5.2?

**Comment by tkelman** [comment]
I'm not going to own testing of any more platforms than I already worry about.

**Comment by ViralBShah** [comment]
Ok - but what is the ETA on 0.5.2 that will include these fixes? Can we do a quick release?

**Comment by tkelman** [comment]
Not without proper testing of anything that goes into it on the platforms people actually use. Can change the download links for these architectures back to 0.5.0 for now.

**Comment by ViralBShah** [comment]
Ok.

**Comment by ppalmes** [comment]
download link for power still points to the binary with libLTO.so issue.

**Comment by ViralBShah** [comment]
@tkelman How does one get 0.5.0?

**Comment by tkelman** [comment]
take the link and replace 0.5.1 with 0.5.0

**Comment by ViralBShah** [comment]
Can you please update it on the downloads page? This is not for me, but for people who don't know how to do it. Even better would be if we could really have 0.5.2 that solves this.

**Comment by tkelman** [comment]
fixed in 0.5.2, reopen or leave a comment if not
